src/pages/software/index.md

```diff
@@ -43,7 +43,7 @@ integration:
       - title: Architecture support
         row:
           - text: X86_64
```

Comment:
Actually @dragetd - looking over this YAML file, it would be super helpful if you could also make the following changes:

- Change `-text: X86_64` to `-text: x86_64` (also a "typo").
- Remove `-text: NEMU`.
- Add `-text: Cloud Hypervisor`.

Entirely optional as I appreciate they are not part of your original PR, but would be most appreciated if you don't mind? :smile:

Reply:
Will do, give me a sec…

Reply:
Done